velodyne_driver/src/driver/driver.cc

```diff
@@ -62,7 +62,12 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
   private_nh.param("model", config_.model, std::string("64E"));
   double packet_rate;                   // packet frequency (Hz)
   std::string model_full_name;
-  if ((config_.model == "64E_S2") || 
+  if ((config_.model == "VLS128") )
+  {
+    packet_rate = 12507;     //https://github.com/velodynevls/velodyne_vls/blob/master/velodyne_driver/src/driver/driver.cc
+    model_full_name = std::string("HDL-") + config_.model;
```

Comment:
Shouldn't this be `VLS-` or just pass in the exact `config._model` value since you're expecting the VLS part to already be there?